examples/python/MPC-spawnrefsmalloffice/main.py

```diff
@@ -17,7 +17,7 @@
 from template_mpc import template_mpc
```

Comment:
I fixed the plotting in this PR. Hopefully this was an issue for you too.

https://github.com/henze-research-group/MODRLC/pull/23

Reply:
I see an issue with the last tick. Hour 24 (or rather 0) does not show. I will add a fix to your PR to show the last tick.